README.md

```diff
@@ -1 +1,48 @@
-# HelloMicroverse
\ No newline at end of file
+### Hello Microverse
+
+    This project is just for Linter learning purpose
+
+### Built With
+
+    HTML
+    CSS
+    Linters
+
+### Live Demo (if available)
+
+    Live Demo Link
+
+### Getting Started
+
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+Authors
+
+👤 Author1
+
+    GitHub: @https://github.com/Hamdullah-Andar
+    Twitter: @https://twitter.com/HamdullahAndar
+    LinkedIn: https://www.linkedin.com/in/hamdullah-andar-98b35894/
+
+🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+Show your support
```

Comment:
- [ ] Kindly update your readme with [this readme template](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md), and make sure to update the readme with your info and include all main headers as stated by the previous reviewer.

Reply:
Hello @osamaashraf6 

I have updated the README file 

I'll be grateful if you could check it and have your feedback

Thank you for the review